Update browser module dblclick docs (#1456)
grafana/k6-docs: docs/sources/next/javascript-api/k6-experimental/browser/element-handle/dblclick.md

---
title: 'dblclick([, options])'
excerpt: 'Browser module: elementHandle.dblclick([, options]) method'
---

# dblclick([, options])

> Mouse double clicks the element.

Reviewer: WDYT?

```suggestion
Mouse double clicks on the element.
```

Author: Resolved in 4dfadddfea4194c7656d8570c63b9089f35cf861

| options             | object   | `null`  |                                                                                                                                                                                                                                                                                                                                               |
| options.button      | string   | `left`  | The mouse button (`left`, `middle` or `right`) to use during the action.                                                                                                                                                                                                                                                                      |
| options.delay       | number   | `0`     | Milliseconds to wait between `mousedown` and `mouseup`.                                                                                                                                                                                                                                                                                       |

> | options.force       | boolean  | `false` | Setting this to `true` will bypass the actionability checks (`visible`, `stable`, `enabled`).                                                                                                                                                                                                                                                 |

Reviewer: Not a strong opinion, but?

```suggestion
| options.force       | boolean  | `false` | Setting this to `true` bypasses the actionability checks (`visible`, `stable`, `enabled`).      
```

Author: Resolved in 0abcf69969a3e825f97446543ddf9575c3bf0ff0

> | options.modifiers   | string[] | `null`  | `Alt`, `Control`, `Meta` or `Shift` modifiers keys pressed during the action. If not specified, currently pressed modifiers are used.                                                                                                                                                                                                         |

Reviewer: What does "currently pressed modifiers" in this context mean?

> If not specified, currently pressed modifiers are used.

Author: If the script has used `keyboard.press('Alt')` before `elementHandle.dblclick()` is used then it will take into account the `Alt` key press.

> | options.noWaitAfter | boolean  | `false` | If set to `true` and a navigation occurs from performing this action, it will not wait for it to complete.                                                                                                                                                                                                                                    |

Reviewer: ```suggestion
| options.noWaitAfter | boolean  | `false` | If set to `true` and navigation occurs while performing this action, it doesn't wait for it to complete.        
```

Author: Resolved in 0abcf69969a3e825f97446543ddf9575c3bf0ff0

| options.position    | object   | `null`  | A point to use relative to the top left corner of the element. If not supplied, a visible point of the element is used.                                                                                                                                                                                                                       |
| options.position.x  | number   | `0`     | The x coordinate.                                                                                                                                                                                                                                                                                                                             |
| options.position.y  | number   | `0`     | The y coordinate.                                                                                                                                                                                                                                                                                                                             |

> | options.timeout     | number   | `30000` | Maximum time in milliseconds. Pass `0` to disable the timeout. Default is overridden by the `setDefaultTimeout` option on [BrowserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/) or [Page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/). |

Reviewer: ```suggestion
| options.timeout     | number   | `30000` | Maximum time in milliseconds. Passing `0` disables the timeout. The `setDefaultTimeout` option on [BrowserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/) or [Page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/) overrides the default timeout. |
```

Author: Resolved in 0abcf69969a3e825f97446543ddf9575c3bf0ff0

> | options.trial       | boolean  | `false` | Setting this to `true` will perform the actionability checks without performing the action.                                                                                                                                                                                                                                                   |

Reviewer: ```suggestion
| options.trial       | boolean  | `false` | Performs the actionability checks without performing the double click action if set to `true`.                                                                                                                                                                                                                                                   |
```

Author: Resolved in 0abcf69969a3e825f97446543ddf9575c3bf0ff0
